### PR TITLE
[themes] Correct the Filters' multiselect item size in Horizon + eslint fixes.

### PR DESCRIPTION
**Pull request by jansiegel**
### Context
This PR:
1. [Modifies the styles for the multiselect component to assign the correct row size.](https://github.com/handsontable/handsontable/commit/75f8b34aded55bda5806887d526f5849d6833807)
2. Configures the `styleslint` script to check both `css` and `scss` files + adds a ruleset for both.
3. Corrects the `css` and `scss` files to match the new `stylelint` rules.
4. Adds a `curly` eslint rule to the entire repo and correct the instances where that rule was not met.

As many of the `scss` files were modified, @adrianspdev, could you check if those changes didn't break anything the themes are meant to do?

### How has this been tested?
Tested locally.

----

This PR is meant to be merged to `feature/dev-issue-1291`.
[skip changelog]

### CODE REVIEW

File: handsontable/src/styles/legacy/handsontable.css [hunk 1]
```diff
@@ -160,14 +158,14 @@
 }
 
 .handsontable th:last-child {
-  /*Foundation framework fix*/
+  /* Foundation framework fi x */
```
Comment (budnix) [inline handsontable/src/styles/legacy/handsontable.css:161]: `fi x`? 😄

File: handsontable/src/styles/legacy/handsontable.css [hunk 2]
```diff
@@ -404,7 +405,7 @@ innerBorderBottom - Property controlled by bottom overlay
 }
 
 /* plugins */
-/* row + column resizer*/
+/* row + column resize r */
```
Comment (budnix) [inline handsontable/src/styles/legacy/handsontable.css:408]:
```suggestion
/* row + column resizer */
```

File: handsontable/src/styles/legacy/handsontable.css [hunk 3]
```diff
@@ -554,13 +555,13 @@ innerBorderBottom - Property controlled by bottom overlay
   font-size: inherit;
   box-shadow: 0 0 0 2px #5292F7 inset;
   resize: none;
-  /*below are needed to overwrite stuff added by jQuery UI Bootstrap theme*/
+  /* below are needed to overwrite stuff added by jQuery UI Bootstrap them e */
```
Comment (budnix) [inline handsontable/src/styles/legacy/handsontable.css:558]:
```suggestion
  /* below are needed to overwrite stuff added by jQuery UI Bootstrap theme */
```

File: handsontable/src/styles/legacy/handsontable.css [hunk 4]
```diff
@@ -554,13 +555,13 @@ innerBorderBottom - Property controlled by bottom overlay
...
   display: block;
   color: #000;
   border-radius: 0;
   background-color: #FFF;
   box-sizing: border-box !important;
-  /*overwrite styles potentionally made by a framework*/
+  /* overwrite styles potentionally made by a framewor k */
```
Comment (budnix) [inline handsontable/src/styles/legacy/handsontable.css:564]:
```suggestion
  /* overwrite styles potentionally made by a framework */
```

File: handsontable/src/styles/legacy/handsontable.css [hunk 5]
```diff
@@ -1890,8 +1897,8 @@ textarea.HandsontableCopyPaste {
   font-size: 8px;
   height: 8px;
   line-height: 1.1;
-  /* Workaround for IE9 - IE11 */
-  text-decoration: underline;
+  /* Workaround for IE9 - IE11, https://stackoverflow.com/a/21902566, https://stackoverflow.com/a/32120247 */
```
Comment (budnix) [inline handsontable/src/styles/legacy/handsontable.css:1900]: Can be safely removed? (not supported browsers)

Comment (jansiegel) [reply handsontable/src/styles/legacy/handsontable.css:1900]: 🤞

**Review by budnix — APPROVED** [review]

**Pull request merged**